core/procedures.js

```diff
@@ -374,7 +374,7 @@ Blockly.Procedures.getPrototypeBlock = function(procCode, workspace) {
 Blockly.Procedures.newProcedureMutation = function() {
   var mutationText = '<xml>' +
       '<mutation' +
-      ' proccode="block name"' +
+      ' proccode="' + Blockly.Msg.PROCEDURE_DEFAULT_NAME + '"' +
```

Comment:
To avoid renames with Closure advanced compilation, prefer `Blockly.Msg['PROCEDURE_DEFAULT_NAME']`, unless you're including translation in your compilation sources.